<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>turn_book</class>
 <widget class="QWidget" name="turn_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Взять книгу</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>691</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ведите  название книги, которую хотите взять:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>831</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ready_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>811</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_start_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
